product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,312 @@
+-   `SSLCertificateKeyFile` is the .key file generated when you created the CSR (for example, your_private.key).
+
+For example, make the following updates:
+
+```shell
+SSLEngine on
+SSLCertificateFile /path/to/your_domain_name.crt
+SSLCertificateKeyFile /path/to/your_private.key
+```
+
+To increase the security, you can replace the HTTPD self-signed SSL certificates with the trusted third-party CA signed certificates in PEM. For more information, see [Replacing HTTPD self-signed SSL certificates](https://www.enterprisedb.com/postgres-tutorials/how-replacing-httpd-self-signed-ssl-certificates-trusted-ca-signed-certificates).
+
+## PEM backend database server and agent connection certificates
+
+PEM, by default, supports secured SSL/TLS connections and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. The self-signed SSL certificates and keys for the PEM server and agent get generated during the PEM installation. These certificates and keys encrypt the connections between the server and the agent. 
+
+To increase the security, you can replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates. For more information, see [Trusted third-party CA certificates and keys](/pem/latest/certificates/#trusted-third-party-ca-certificates-and-keys).
+
+### How PEM self-signed SSL certificates work
+
+The PEM server configuration script generates self-signed SSL certificates and key files for the PEM backend database server. The backend database server uses these certificates and keys to authenticate and encrypt the agent connections. Each certificate has an expiry date. You should regenerate the certificates when it nears expiration. For more information, see [Regenerating server self-signed SSL certificates](/pem/latest/certificates/replacing_ssl_certificates).
+
+The PEM agent connects to the PEM backend database server using the libpq interface, acting as a client of the backend database server. The agent self-signed SSL certificates and keys get generated during the [PEM registration](/pem/latest/registering_agent). PEM agent establishes the connection with the PEM backend database server using the self-signed SSL certificate and key files. 
+
+Each agent has a unique identifier and the agent certificates and keys have the corresponding identifier mentioned. Each certificate has an expiry date. You should regenerate the certificates when it nears expiration. For more information, see [Regenerating agent self-signed SSL certificates](/pem/latest/certificates/regenerating_agent_certificates).
+
+You can generate a common agent certificate and key file pair for all the PEM agents and use for connecting to PEM backend database server. For more information see [Generate common agent certificate and key pair](/pem/lates/certificates/#generate-a-common-agent-certificate-and-key-pair)
```

Comment:
This might be worth drawing more attention to and explaining why you might do it.